client/my-sites/woocommerce/dashboard/required-plugins-install-view.jsx

```diff
@@ -535,7 +547,13 @@ class RequiredPluginsInstallView extends Component {
 						title={ translate( "We can't update your store" ) }
 						subtitle={ subtitle }
 					>
-						<Button primary href={ CALYPSO_CONTACT } target="_blank" rel="noopener noreferrer">
+						<Button
+							onClick={ this.trackContactSupport( 'failure' ) }
```

Comment:
This calls `trackContactSupport()` immediately when the contact support message is rendered.

What you want is something like...

```
onClick={ () => { this.trackContactSupport( 'failure' ) } }
```

or

```
onClick={ this.trackContactSupport.bind( this, 'failure' ) }
```

They both create a new function every time the message is rendered. I'd go with the anonymous arrow function, since I think it is more readable and familiar to newer JS devs.

Reply:
Ah, rookie mistake! Thanks for catching this